src/flexmeasures_client/client.py

```diff
@@ -46,10 +47,20 @@ class FlexMeasuresClient:
     session: ClientSession | None = None
 
     def __post_init__(self):
-        if not self.scheme:
-            self.scheme: str = "http" if "localhost" in self.host else "https"
-        if self.ssl is None:
-            self.ssl: bool = False if "localhost" in self.host else True
+        if not re.match(r".+\@.+\..+", self.email):
+            print(self.email)
```

Comment:
I would remove this print.

Reply:
Thanks for the review. removed the print. Logging will be a later issue #14